java/jdbc/src/main/java/io/vitess/jdbc/VitessPreparedStatement.java

```diff
@@ -141,18 +136,17 @@ public ResultSet executeQuery() throws SQLException {
                             .streamExecute(context, this.sql, this.bindVariables, tabletType, vitessConnection.getIncludedFields());
```

Comment:
@harshit-gangal this is taken care of in https://github.com/youtube/vitess/pull/2943 so this change now looks good